MEIOUandTaxes1/src/events/DG-01-Papacy.txt

```diff
@@ -6006,7 +6006,7 @@ country_event = {
 	}
 	option = {
 		name = "DG_Papacy.901.opta"
-		add_reform_desire = 0.00002
+		add_reform_desire = 0.00001
```

Comment:
I am not sure such small values actually do anything. AFAIK most eu4 effects if not all take at max 3 decimal places. This is not an issue with this merge but generally with the DG code